.vsts-ci.yml

```diff
@@ -112,11 +111,16 @@ extends:
               inputs:
                 sourceFolder: 'artifacts/packages/$(_BuildConfig)/Shipping/'
                 contents: '*.msi'
-                targetFolder: '$(Build.ArtifactStagingDirectory)'
+                targetFolder: '$(Build.ArtifactStagingDirectory)\artifacts'
+            - task: MicroBuildCodesignVerify@3
+              inputs:
+                TargetFolders: '$(Build.ArtifactStagingDirectory)\artifacts'
+                ExcludeSNVerify: true
```

Comment:
What does ExcludeSNVerify do? May you provide the link to where this task is documented, I couldn't find it on the web.

Reply:
Sure: https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/653/Codesign-Verification